fw/http_parser.c

```diff
@@ -10683,7 +10683,7 @@ tfw_h2_parse_body(char *data, unsigned int len, TfwHttpReq *req,
 	       parser->to_read, m_len, len);
 
 	if (tfw_http_msg_add_str_data(msg, &req->body, data, m_len))
-		return T_DROP;
+		return T_BLOCK;
```

Comment:
For T_DROP we don't close connection if block_action error reply, but we can't communicate with this client because of hpack dynamic table difference for client and server. But I agree, we should return T_DROP and close connection with FIN. We should implement such behavior for all cases where we can't work with this client, but no attack detected.